tests/test_database_connector/test_check_data.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from datetime import datetime
+
+from sensorflux.database_connector import DatabaseConnector
+
+
+def test_check_data():
+    """
+    GIVEN data to be sent to influxdb
+    WHEN the method is called with data as an argument
+    THEN it should only return True if the data is valid
+    """
+    connector = DatabaseConnector(
+        measurement='measurement',
+        device='device',
+        fields=('temp', 'atmo', 'humi'))
+
+    assert connector.check_data({
+        'time': datetime.utcnow().isoformat(),
+        'temp': 123,
+        'atmo': 456,
+        'humi': 789}) is True
+    assert connector.check_data({
+        'time': datetime.utcnow().isoformat(),
+        'humi': 789}) is True
+    assert connector.check_data({'temp': 123, 'atmo': 789}) is True
+    assert connector.check_data({}) is False
```

Comment:
Here instead you could write
`assert not connector.check_data({})`
It's just more concise and "natural"